.travis.yml

```diff
@@ -32,6 +36,7 @@ cache:
     - $HOME/.composer/cache
 
 before_install:
+  - if [[ -v $STABILITY ]]; then composer config minimum-stability $STABILITY; fi;
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
```

Comment:
this line must be updated, as you replaced `SYMFONY_VERSION=2.8.*` by `DEPENDENCIES="symfony/lts:^2"` in the job matrix and this is the corresponding one